skaffold/src/test/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurationsTest

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.skaffold.run
+
+import com.google.common.truth.Truth
+import com.google.container.tools.core.PluginInfo
+import com.google.container.tools.skaffold.SkaffoldExecutorService
+import com.google.container.tools.skaffold.SkaffoldExecutorSettings
+import com.google.container.tools.skaffold.SkaffoldProcess
+import com.google.container.tools.test.ContainerToolsRule
+import com.google.container.tools.test.TestService
+import com.google.container.tools.test.expectThrows
+import com.intellij.execution.ExecutionException
+import com.intellij.execution.RunnerAndConfigurationSettings
+import com.intellij.execution.configurations.RunConfigurationBase
+import com.intellij.execution.configurations.SearchScopeProvider
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.project.guessProjectDir
+import io.mockk.CapturingSlot
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.slot
+import org.jdom.Element
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.io.File
+
+/** Unit tests for [SkaffoldCommandLineState] */
+class SkaffoldCommandLineStateTest {
```

Comment:
the class name should typically match the file name. did you mean to call this class `SkaffoldRunConfigurationsTest`?

Reply:
also it looks like you added it as a text file instead of a `.kt` file. 

When creating a new Kotlin file in the IDE, use:
`right click on the folder` -> new -> Kotline File/class